src/olympia/files/utils.py

```diff
@@ -251,14 +255,18 @@ def apps(self):
         )
 
         for app, default_min_version in apps:
-            strict_min_version = (
-                # At least this version supports installing.
-                get_simple_version(self.gecko.get('strict_min_version')) or
-                default_min_version)
+            if self.guid is None:
+                strict_min_version = amo.DEFAULT_WEBEXT_MIN_VERSION_NO_ID
```

Comment:
So, if no `id` is given, you're overriding the `strict_min_version` no matter what ? What if it was specifying a higher version ? It may not matter right now because we don't have releases past 48.0 yet, but this could be a problem in the future.